src/pages/chan/components/Comment/Comment.js

```diff
@@ -0,0 +1,57 @@
+import { React, useState } from 'react';
+import CommentList from '../CommentList/CommentList';
+import './Comment.scss';
+
+function Comment() {
+  const [userName] = useState('JinHee');
+  const [comment, setComment] = useState('');
+  const [feedCommets, setFeedComments] = useState([]);
+  const [isValid, setIsValid] = useState(false);
```

Comment:
isValid도 위 코멘트와 같은 맥락으로 state로 두지 않아도 되는 변수입니다!

how about
```suggestion
  const isFilled = comment.length !== 0  ;
```